sycl/test-e2e/Matrix/SG32/joint_matrix_prefetch.cpp

```diff
@@ -10,7 +10,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: gpu
+// XFAIL: gpu && !gpu-intel-dg2
```

Comment:
Tests in SG32 folder are not supported on DG2. This is how we exclude DG2 in all other tests in SG32 folder:

```suggestion
// XFAIL: gpu

// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
// UNSUPPORTED: gpu-intel-dg2
```

Reply:
Will fix in follow-up PR immediately

Reply:
https://github.com/intel/llvm/pull/14870